### PR TITLE
Add `supports_attributes` property and Label Studio docs

### PR DESCRIPTION
**Pull request by ehofesmann**
Hi @Rusteam ! 

This PR adds the `supports_attributes` property to annotation backends and will raise a warning when a backend like label studio does not support attributes, but they were provided anyway. There are a couple of other tweaks made to the label studio code to support the `media_field` argument and load credentials properly.

This also adds an integration page for Label Studio similar to the CVAT and Labelbox integrations.

Let me know if you have any questions/thoughts about any of this! After this is merged, we should be just about ready to get your PR merged into FiftyOne.

### CODE REVIEW

File: fiftyone/utils/labelstudio.py [hunk 1]
```diff
@@ -132,6 +132,10 @@ def supports_keyframes(self):
     def supports_video_sample_fields(self):
         return False
 
+    @property
+    def supports_attributes(self):
+        return False
```
Comment (Rusteam) [inline fiftyone/utils/labelstudio.py:138]: thanks for that, I forgot about it :)

File: docs/source/integrations/labelstudio.rst [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+========================
+
+.. default-role:: code
+
+`Label Studio <https://labelstud.io/>`_ is a sleek and popular open-source data
+labelling tool with a simple and straightforward UI. The integration between
+FiftyOne and Label Studio allows you to easily upload your data directly from
+FiftyOne to Label Studio for labeling.
+
+You can get start with Label Studio through a simple pip install to get a local
```
Comment (Rusteam) [inline docs/source/integrations/labelstudio.rst:13]: started

File: fiftyone/utils/labelstudio.py [hunk 3]
```diff
@@ -243,17 +242,17 @@ def _init_project(self, config, samples):
         )
         return project
 
-    def _prepare_tasks(self, samples, label_schema):
+    def _prepare_tasks(self, samples, label_schema, media_field):
```
Comment (Rusteam) [inline fiftyone/utils/labelstudio.py:245]: thanks for handling this

File: docs/source/integrations/labelstudio.rst [hunk 4]
```diff
@@ -0,0 +1,631 @@
...
+server up and running. FiftyOne provides
+:ref:`simple setup <label-studio-setup>` instructions
+that you can use to specify the necessary account credentials and server
+endpoint to use.
+
```
Comment (Rusteam) [inline docs/source/integrations/labelstudio.rst:18]: Many thanks for this write-up! I can't imagine how long it would take myself to do something similar.

**Review by Rusteam — APPROVED** [review]

File: docs/source/integrations/labelstudio.rst (hunk 2 above)
Comment (ehofesmann) [reply docs/source/integrations/labelstudio.rst:13]: Good catch!

File: docs/source/integrations/labelstudio.rst (hunk 4 above)
Comment (ehofesmann) [reply docs/source/integrations/labelstudio.rst:18]: Haha, of course!

**Pull request merged**